infra/modules/setup/aws_auth.tf

```diff
@@ -20,7 +20,12 @@ resource "kubernetes_config_map" "aws_auth" {
   groups:
     - system:masters
 EOF
-
+    mapUsers = <<EOF
+- userarn: ${var.user_role_arn}
```

Comment:
This variable is wrong, it is currently pointing to the user role arn, this will need to be an additional variable passed through that is the arn for the service user you have also created.